packages/components/src/internal/components/base/Grid.tsx

```diff
@@ -109,7 +110,7 @@ function processColumns(columns: List<any>): List<GridColumn> {
                 align: c.align,
                 cell: c.cell,
                 format: c.jsonType === 'float' || c.jsonType === 'int' ? c.format : undefined,
-                index: c.index || c.fieldKey,
+                index: c.index || QueryKey.decodePart(c.fieldKey),
```

Comment:
I presume the correlated row data is already composed using the decoded `fieldKey`? 

I'd prefer to leave `Grid` unaware of `@labkey/api` if we can and work out the decoding alignment with the data at a higher point where we can decode all column field keys (possibly even in the `getQueryDetails.api` response).

Reply:
It looks like the QueryColumn model already has a second/alternate fieldKey property with the decoded value, so I'll switch to using that instead of decoding here. Thanks.